src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,28 @@
+.calendar {
+  padding: 10px;
+  width: #{$limit-width-of-row};
+  display: flex;
+  gap: $indent-of-days;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    text-decoration: none;
+
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+
+    align-items: center;
+    justify-content: center;
```

Comment:
The `align-items: center;` and `justify-content: center;` properties are used with a flex container but you haven't declared `display: flex;` or `display: grid;` for the `&__day` element. These properties won't have any effect without it.